tests/NLog.UnitTests/Targets/NetworkTargetTests.cs

```diff
@@ -93,20 +94,35 @@ public void NetworkTargetHappyPathTest()
             var sender = senderFactory.Senders[0];
             target.Close();
 
-            Assert.Equal(18L, sender.MemoryStream.Length);
-            Assert.Equal("msg1\r\nmsg2\r\nmsg3\r\n", target.Encoding.GetString(sender.MemoryStream.GetBuffer(), 0, (int)sender.MemoryStream.Length));
+            Assert.Equal(15L, sender.MemoryStream.Length);
```

Comment:
Don't change current tests! Create a new one (with exacted method to avoid too much duplication), or make a parametrised test (not supported on silverligth)